core/src/model/fact.rs

```diff
@@ -265,7 +265,7 @@ impl Fact for TypedFact {
 
     fn matches(&self, t: &Tensor, symbols: Option<&SymbolValues>) -> TractResult<bool> {
         let shape = self.shape.eval_to_usize(symbols.unwrap_or(&SymbolValues::default()))?;
-        Ok(self.datum_type == t.datum_type() && &**shape == t.shape())
+        Ok(self.datum_type.unquantized() == t.datum_type().unquantized() && &**shape == t.shape())
```

Comment:
Some operators operate on U8 or I8 instead of the quantized type, as there is no difference (MaxPool, for example). Should I fix this so the types match everywhere, or leave it like that ?